indico/modules/events/papers/forms.py

```diff
@@ -94,6 +94,11 @@ class PaperReviewingSettingsForm(IndicoForm):
     hide_accepted = BooleanField(_('Keep papers hidden'), widget=SwitchWidget(),
                                  description=_("Keep papers hidden from participants even after they've "
                                                'been accepted.'))
+    enforce_evaluation_rounds = BooleanField(_('Enforce evaluation rounds'), widget=SwitchWidget(),
```

Comment:
I think we can always enforce if a limit is set. For the deadlines it makes more sense to have it separate (because even before reaching it you already see the deadline), but for the number of review rounds I'm not sure the same applies...